audit-cli/README.md

```diff
@@ -222,7 +223,9 @@ With `-v` flag, also shows:
 
 #### `analyze includes`
 
-Analyze `include` directive relationships in RST files to understand file dependencies.
+Analyze `include` directive and `toctree` relationships in RST files to understand file dependencies.
```

Comment:
Oh, this is an interesting interpretation.

I would prefer this did not include `toctree` relationships by default. Can we make this an optional flag? I was specifically trying to inspect all the content that goes into creating _this_ file or page. The toctree references are links to other pages, but they are not pulled into the page content (transcluded) the same way that includes are.